perf/micro/immediate-scheduler/observable/from-array.js

```diff
@@ -8,7 +8,7 @@ module.exports = function fromArray(suite) {
     }
 
     var oldFromArrayWithImmediateScheduler = RxOld.Observable.fromArray(args, RxOld.Scheduler.immediate);
-    var newFromArrayWithImmediateScheduler = RxNew.Observable.fromArray(args);
+    var newFromArrayWithImmediateScheduler = RxNew.Observable.from(args);
```

Comment:
It seems makes sense. I'm also quite curious about why Array.isArray is taking those noticeable amount of time, still not sure though.